cluster/gce/gci/configure-helper.sh

```diff
@@ -1338,7 +1338,7 @@ function assemble-docker-flags {
   if [[ "${TEST_CLUSTER:-}" == "true" ]]; then
     docker_opts+=" --log-level=debug"
   else
-    docker_opts+=" --log-level=warn"
+    docker_opts+=" --log-level=info"
```

Comment:
what is the increase in log volume due to this?

Reply:
Have you looked at https://github.com/kubernetes/kubernetes/issues/27584? We used to have it at info and changed it to warn.

Reply:
Thanks @liggit and @cheftako. There's some more info in #15590 and #15728. Those issues pretty old (from 2016), so let me take a closer look at current docker logging volume at info compared to warn today.